app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt

```diff
@@ -414,6 +416,14 @@ class Message : RealmObject, Snoozable {
         snoozeEndDate = flags.snoozeEndDate.toRealmInstant()
     }
 
+    fun getFormattedPreview(context: Context): String {
+        return when {
```

Comment:
We like to put the when as expression body when the only thing the function does is returning a when. 
It saves a level of indentation and stays easily readable.

```suggestion
    fun getFormattedPreview(context: Context) = when {
```